packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -93,14 +94,30 @@ export class MirrorNodeClient {
     protected createAxiosClient(
         baseUrl: string
     ): AxiosInstance {
-        return Axios.create({
+        const axiosClient: AxiosInstance = Axios.create({
             baseURL: baseUrl,
             responseType: 'json' as const,
             headers: {
                 'Content-Type': 'application/json'
             },
             timeout: 10 * 1000
         });
+
+        // @ts-ignore
+        axiosRetry(axiosClient, {
+            retries: 10,
```

Comment:
nit: 10 might be a lot in production.
In test cases fine. 3 or 5 probably max for now
```suggestion
            retries: 5,
```